x/ccv/provider/keeper/relay.go

```diff
@@ -34,14 +34,30 @@ func (k Keeper) OnRecvVSCMaturedPacket(
 	packet channeltypes.Packet,
 	data ccv.VSCMaturedPacketData,
 ) exported.Acknowledgement {
-	// check that the channel is established
-	chainID, found := k.GetChannelToChain(ctx, packet.DestinationChannel)
-	if !found {
-		// VSCMatured packet was sent on a channel different than any of the established CCV channels;
-		// this should never happen
-		panic(fmt.Errorf("VSCMaturedPacket received on unknown channel %s", packet.DestinationChannel))
+
+	if err := k.ValidateVSCMaturedPacket(ctx, packet, data); err != nil {
+		return channeltypes.NewErrorAcknowledgement(err.Error())
```

Comment:
This never happens

Reply:
True, I can remove the unused path if https://github.com/cosmos/interchain-security/pull/542#discussion_r1038339867 should not be validated